hangman.js

```diff
@@ -0,0 +1,77 @@
+document.addEventListener("DOMContentLoaded", () => {
+    
+const celebrities = ['Beyonce', 'Taylor Swift', 'Ellen DeGeneres', 'Dwayne Johnson', 'George Clooney', 'Sean Combs', 'Alec Baldwin', 'Bradley Cooper', 'Serena Williams', 'Laila Ali', 'Rosario Dawson', 'Belcalis Almanzar', 'Leonardo DiCaprio', 'Liam Hemsworth', 'Will Smith', 'Shawn Carter', 'Sofia Vergara', 'Demi Lovato', 'Jennifer Lawrence','Miley Cyrus','Idris Elba', 'Kanye West','Kylie Jenner','Ed Sheeran','Rihanna','Drake','Ermias Asghedom', 'Chris Brown', ]
+
+const celebWord = (celebrities) => {
+    return celebrities[Math.floor(Math.random() * celebrities.length)];
+} 
+let hiddenCeleb = celebWord(celebrities);
+
+
+const alphabet = new Set('abcdefghijklmnopqrstuvwxyz');
+let guessedLetters = new Set();
+
+const selectCeleb = (word) => {
+    let displayCelebWord = document.querySelector("h3");
+    let blanks = ('_ ').repeat(selectCeleb.length);
+    displayCelebWord.innerText = blanks;
+}
+selectCeleb(hiddenCeleb) ;
+
+let form = document.querySelector('form')
+  form.addEventListener ("submit", (event) => {
+    event.preventDefault();
+
+let winner = false;
+let misses = 6;
+
+while (!winner && misses > 0) {
+    console.log(`you have ${misses} guesses left`);
+    console.log(`Guessed letters: ${Array.from(guessedLetters).join(', ')}`);
+    console.log(blanks);
+  
+  
```

Comment:
Seems like everything happens in the console, instead of the actual webpage document.